exporter/alibabacloudlogserviceexporter/metricsdata_to_logservice.go

```diff
@@ -290,13 +281,9 @@ func metricDataToLogServiceData(md pdata.Metric, defaultLabels KeyValues) (logs
 	switch md.DataType() {
 	case pdata.MetricDataTypeNone:
 		break
-	case pdata.MetricDataTypeIntGauge:
-		return intMetricsToLogs(md.Name(), md.IntGauge().DataPoints(), defaultLabels)
-	case pdata.MetricDataTypeGauge:
+	case pdata.MetricDataTypeIntGauge, pdata.MetricDataTypeGauge:
```

Comment:
Why "casing" on the `pdata.MetricDataTypeIntGauge`?

Reply:
Removed.